### DISCUSSION TITLE
Mandarin V language discussing, 我建了个QQ群 830827849, 方便中文交流学习V语言

### DISCUSSION THREAD

**Discussion by ToughStyle** [vlang/v#18209] [General]
加入QQ群学习讨论更方便, V语言中文交流群: 830827849

**Comment by Jengro777** [comment]
为什么不是微信群？

QQ   已经很久很久不用了

**Reply by flamehe0580** [reply]
似乎没见过有用微信做技术交流群的。微信连群文件都不支持，微信发个视频还被模糊处理得一塌糊涂。

**Reply by Jengro777** [reply]
企业微信  
你不用不代表别人不用，我们反而觉得QQ 不怎么使用了

**Comment by felixf4xu** [comment]
website?

**Reply by tzSharing** [reply]
A popular instant messaging software in China
